kiwi/system/root_bind.py

```diff
@@ -82,6 +83,11 @@ def mount_kernel_file_systems(self) -> None:
             fails to mount
         """
         try:
+            shared_mount = MountManager(
+                device=self.root_dir, mountpoint=self.root_dir
```

Comment:
I don't understand that...

Reply:
yes sorry my fault, this cannot be understood without an explanation. The way this mount is done is a trick to overcome an issue in docker and podman when called from within a chroot. There is some information in this thread

* https://github.com/moby/moby/issues/34817

I think a comment with that issue link is a must have

Reply:
done